kubernetes/resource_kubernetes_pod_test.go

```diff
@@ -2346,3 +2330,18 @@ resource "kubernetes_config_map" "test_from" {
 }
 `, provider, name, imageName)
 }
+
+func testAccKubernetesPodConfigMinimal(name string) string {
+	return fmt.Sprintf(`resource "kubernetes_pod" "test" {
+  metadata {
+    name = "%s"
+  }
+  spec {
+    container {
+        image = "busybox"
+        name  = "containername"
+    }
+  }
+}
+`, name)
+}
```

Comment:
The idea with this test is to check that our defaults are being set correctly. I was finding that most our tests are explicitly setting attributes like `automount_service_account_token`, but when you leave that attribute out of your config, it caused an non-empty plan. So I wanted to check all our defaults relating to PodSpec.